packages/mobile/src/components/lineup/Lineup.tsx

```diff
@@ -526,7 +527,11 @@ export const Lineup = ({
           hideHeaderOnEmpty && areSectionsEmpty ? undefined : header
         }
         ListFooterComponent={
-          lineup.hasMore ? <View style={{ height: 16 }} /> : ListFooterComponent
+          lineup.hasMore ? (
+            <View style={{ height: 16 }} />
```

Comment:
spacing?

Reply:
not exactly sure why that is there, but yeah assume it's spacing :)